### ISSUE TITLE
contactgroups split error

### ISSUE THREAD

**Issue by huasome** [naparuba/shinken#1447]
Try to use comma separated contactgroups in contacts in version 2.2rc1 and getting

```
[1420845444] CRITICAL: [Shinken] Back trace of the error: Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/shinken/daemons/arbiterdaemon.py", line 590, in main
    self.load_config_file()
  File "/usr/local/lib/python2.7/dist-packages/shinken/daemons/arbiterdaemon.py", line 385, in load_config_file
    self.conf.explode()
  File "/usr/local/lib/python2.7/dist-packages/shinken/objects/config.py", line 1059, in explode
    self.contacts.explode(self.contactgroups, self.notificationways)
  File "/usr/local/lib/python2.7/dist-packages/shinken/objects/contact.py", line 267, in explode
    for cg in c.contactgroups.split(','):
AttributeError: 'list' object has no attribute 'split'
```

**Comment by david-guenault** [comment]
could you provide some sample configuration in order to be able to reproduce ?

**Comment by naparuba** [comment]
Any news? we will release the 2.2 so if we can have more information before will be cool.

Thanks :)

**Comment by huasome** [comment]
Oh, sorry, the syntax looks like this:

```
define contactgroup {
   contactgroup_name cg_admins
   alias                cg_admins
}

define contactgroup {
    contactgroup_name   cg_support
    alias               cg_support
}

define contact {
    use             generic-contact
    contact_name    foo
    alias           Foo Bar
    contactgroups   cg_admins,cg_itsupport
    email           foo@sample.com
    pager          415-555-5555
    is_admin        1
}
```

**Comment by Seb-Solon** [comment]
I can confirm I have no such stack trace on upstream version with the provided file.

**Comment by david-guenault** [comment]
It seem it happen randomly on my lab. Working on it
Le 12 janv. 2015 20:12, "Sébastien Coavoux" <notifications@github.com> a
écrit :

**Comment by david-guenault** [comment]
Some time at startup in a HA setup i can see this kind of stacktrace ... but i'm not able to reproduce it everytime. Yes it also sound strange for me but the fact is that i saw it so i think we must dig into this problem.

**Comment by david-guenault** [comment]
With the sample provided i've been able to reproduce

```
  File "/usr/lib/python2.6/site-packages/Shinken-2.2_RC1-py2.6.egg/shinken/daemons/arbiterdaemon.py", line 590, in main
    self.load_config_file()
  File "/usr/lib/python2.6/site-packages/Shinken-2.2_RC1-py2.6.egg/shinken/daemons/arbiterdaemon.py", line 385, in load_config_file
    self.conf.explode()
  File "/usr/lib/python2.6/site-packages/Shinken-2.2_RC1-py2.6.egg/shinken/objects/config.py", line 1059, in explode
    self.contacts.explode(self.contactgroups, self.notificationways)
  File "/usr/lib/python2.6/site-packages/Shinken-2.2_RC1-py2.6.egg/shinken/objects/contact.py", line 267, in explode
    for cg in c.contactgroups.split(','):
AttributeError: 'list' object has no attribute 'split'
```

**Comment by david-guenault** [comment]
seems to be fixed in master branch.

**Comment by david-guenault** [comment]
Fixed in upstream (see #1425)
